torch/utils/tensorboard/utils.py

```diff
@@ -0,0 +1,123 @@
+# Functions for converting
```

Comment:
This is a private utility module right? Prefix it with an underscore if so.

Reply:
renamed to _utils.py